### PR TITLE
Log StructuredLog: spelling formatting

### PR DESCRIPTION
**Pull request by heqg**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
According to https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
Capitalize the first letter of the sentence

Which issue(s) this PR fixes:
Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?:
```release-note
NONE
```

Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

### CODE REVIEW

File: cmd/kubelet/app/server_linux.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func watchForLockfileContention(path string, done chan struct{}) error {
 	go func() {
 		select {
 		case ev := <-watcher.Event:
-			klog.InfoS("inotify event", "event", ev)
+			klog.InfoS("Inotify event", "event", ev)
 		case err = <-watcher.Error:
 			klog.ErrorS(err, "inotify watcher error")
```
Comment (wangyysde) [inline cmd/kubelet/app/server_linux.go:42]: should be capitalized for this line?

File: pkg/volume/local/local.go [hunk 2]
```diff
@@ -412,7 +412,7 @@ func (plugin *localVolumePlugin) NodeExpand(resizeOptions volume.NodeResizeOptio
 	case hostutil.FileTypeDirectory:
 		// if the given local volume path is of already filesystem directory, return directly because
 		// we do not want to prevent mount operation from succeeding.
-		klog.InfoS("expansion of directory based local volumes is NO-OP", "local-volume-path", localDevicePath)
+		klog.InfoS("Expansion of directory based local volumes is NO-OP", "local-volume-path", localDevicePath)
 		return true, nil
 	default:
 		return false, fmt.Errorf("only directory and block device are supported")
```
Comment (wangyysde) [inline pkg/volume/local/local.go:418]: should be capitalized for this line?

File: cmd/kubelet/app/server_linux.go (hunk 1 above)
Comment (heqg) [reply cmd/kubelet/app/server_linux.go:42]: According to https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
Capitalize the first letter of the sentence

File: pkg/volume/local/local.go (hunk 2 above)
Comment (heqg) [reply pkg/volume/local/local.go:418]: According to https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
Capitalize the first letter of the sentence

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**